core/src/main/scala/org/genivi/sota/core/VehicleUpdatesResource.scala

```diff
@@ -183,7 +183,9 @@ class VehicleUpdatesResource(db : Database, resolverClient: ExternalResolverClie
       put {
         path("installed") { updateInstalledPackages(vin) } ~
         path("order") { setInstallOrder(vin) } ~
-        (extractUuid & path("cancelupdate") ) { uuid => cancelUpdate(vin, uuid) }
+        (extractUuid & path("cancelupdate") ) { uuid =>
+          namespaceExtractor { ns => cancelUpdate(ns, vin, uuid) }
```

Comment:
If we provide a namespace it should be a `Namespace` and not an Option. But if we can implement something without the namespace, that would be better as this leaves less room for problems and sometimes the namespace is not available, for example if the client is making the request.